webui/src/lib/components/repositoryCreateForm.jsx

```diff
@@ -63,20 +71,26 @@ export const RepositoryCreateForm = ({ config, onSubmit, onCancel, error = null,
             <Form.Group as={Row} controlId="id">
                 <Form.Label column sm={fieldNameOffset}>Repository ID</Form.Label>
                 <Col sm={sm}>
-                    <Form.Control type="text" autoFocus ref={repoNameField} onChange={checkRepoValidity}/>
-                    {!repoValid &&
+                    <Form.Control type="text" autoFocus ref={repoNameField} onChange={onRepoNameChange}/>
+                    {repoValid === false &&
```

Comment:
```suggestion
                    {!repoValid &&
```
(think I prefer this.)

Reply:
This is actually fixing a bug where the "Create repository" button becomes enabled even if you don't fill the storage namespace.